health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/StackedTable.js

```
@@ -0,0 +1,237 @@
+
+  const StackedRowV2 = (rowData) => {
+    const values = rowData?.values;
+    const valueKeys = Object.keys(values);
+
+    return (
+      <div className="digit-stacked-row">
+        <div className="digit-stacked-row-index">{getTitleHeading(rowData?.name)}</div>
+        {valueKeys?.map((key, index) => {
+          const valueItem = values[key];
+          return (
+            <React.Fragment>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)